Jenkinsfile

```diff
@@ -147,12 +147,20 @@ pipeline {
                 KONG_BUILD_TOOLS_LOCATION = "${env.WORKSPACE}/../kong-build-tools"
                 BINTRAY_USR = 'kong-inc_travis-ci@kong'
                 BINTRAY_KEY = credentials('bintray_travis_key')
+                AWS_ACCESS_KEY = credentials('AWS_ACCESS_KEY')
+                AWS_SECRET_ACCESS_KEY = credentials('AWS_SECRET_ACCESS_KEY')
+                CACHE = false
+                UPDATE_CACHE = true
                 DEBUG = 0
+                RELEASE_DOCKER_ONLY=true
+                PACKAGE_TYPE=apk
+                RESTY_IMAGE_BASE=alpine
+                RESTY_IMAGE_TAG=latest
             }
             steps {
                 sh 'echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin || true'
                 sh 'make setup-kong-build-tools'
-                sh 'KONG_VERSION=`git rev-parse --short HEAD` RELEASE_DOCKER_ONLY=true PACKAGE_TYPE=apk RESTY_IMAGE_BASE=alpine RESTY_IMAGE_TAG=latest make release'
+                sh 'KONG_VERSION=`git rev-parse --short HEAD` DOCKER_MACHINE_ARM64_NAME="jenkins-kong-"`cat /proc/sys/kernel/random/uuid` make release'
```

Comment:
Could the `docker-machine rm` command be failing and causing the dangling keys?

Reply:
Yes. There's a variety of reasons
- humans testing locally use the same name
- builds fail / timeout / don't do their cleanup
- temporary credentials fail and cleaning up isn't possible.

For these reasons we do have a lambda that nukes ec2's / instances after 3 hours but yeah there's enough churn we need unique names

Reply:
Got it. Did not know about the lambda! Nice!